lib/commands/option-model.json

```diff
@@ -65,6 +65,12 @@
     "alias": null,
     "stringInput": "REQUIRED"
   },
+  "templateBranch": {
+    "name": "template-branch",
+    "description": "Git branch used with the template url",
+    "alias": null,
+    "stringInput": "OPTIONAL"
```

Comment:
stringInput is required for this branch, if it's optional, then --template-branch without any value is allowed as well.

Reply:
Oh ok. it is little confusing. I will fix. it.